airflow/dags/parse_elavon/elavon_to_gcs_jsonl.py

```diff
@@ -26,19 +26,22 @@ def fetch_and_clean_from_gcs():
     all_rows = pd.DataFrame()
 
     # List raw files available from GCS
-    gfs = gcsfs.GCSFileSystem(
-        project="cal-itp-data-infra",
-        token=BIGQUERY_KEYFILE_LOCATION,
-    )
-    file_list = gfs.ls("test-calitp-elavon-raw/", detail=False)
+    file_and_dir_list = fs.ls("test-calitp-elavon-raw/", detail=False)
+    dir_list = [x for x in file_and_dir_list if fs.isdir(x)]
+
+    # Drill down to the latest export (folders are "ts=" format)
+    target_dir = max(dir_list)
+    file_list = fs.ls(f"{target_dir}/", detail=False)
```

Comment:
If this runs daily, it should probably process that day's extracted data rather than the latest (e.g. what happens if it fails and another sync ran before this is re-run)

Reply:
Discussed offline, I'm fine with leaving it for now.